### PR TITLE
Feat: Add new parser PostfixMaster

### PR DESCRIPTION
**Pull request by TZ3070**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add new parser postfix_conf for adviser rule

### CODE REVIEW

File: insights/parsers/postfix_conf.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        24
+        >>> postfix_master.data[-1] == {'service': 'scache', 'type': 'unix', 'private': '-', 'unpriv': '-', 'chroot': 'n', 'wakeup': '-', 'maxproc': '1', 'command': 'scache'}
+        True
+
+    Raises:
+        insights.core.exceptions.SkipComponent: if the ``/etc/postfix/master.cf`` is empty.
+    """
+
+    def parse_content(self, content):
+        self.data = []
```
Comment (xiangce) [inline insights/parsers/postfix_conf.py:64]: `.data` is not suggested to use.

File: insights/parsers/postfix_conf.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+"""
+from insights.core import ConfigParser
+from insights.core.exceptions import SkipComponent
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.postfix_master)
+class PostfixMaster(ConfigParser):
```
Comment (xiangce) [inline insights/parsers/postfix_conf.py:13]: I'm wondering why `ConfigParser` is used for this parser, as I didn't check that the `parser_doc` is implemented and the super `parser_conent` is called.

File: insights/parsers/postfix_conf.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
+"""
+PostfixMaster - file ``/etc/postfix/master.cf``
```
Comment (xiangce) [inline insights/parsers/postfix_conf.py:2]: Please double confirm the title and the name of the python file. Did you use different names on purpose?

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/postfix_conf.py (hunk 1 above)
Comment (TZ3070) [reply insights/parsers/postfix_conf.py]: Updated

**Review by xiangce — APPROVED** [review]

**Pull request merged**